recipe/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "sis-extras" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/sis_extras-{{ version }}.tar.gz
+  sha256: e7e1892ab5eef36e38880dc9b636597dce83162df4d51f3f7f509a7e792e8a73
+
+build:
+  skip: True  # [py>38]
+  # no streamlit on s390x
+  skip: True  # [s390x]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pdm-backend
+    - pip
+  run:
+    - python
+    - streamlit # ==1.31.1
+    - snowflake-snowpark-python >=1.13.0
+    - plotly >=5.20.0
+
+test:
+  source_files:
+    - tests
+  imports:
+    - sis_extras
+  requires:
+    - pip
+    - pytest
+  commands:
+    #- pip check
```

Comment:
Why is this commented out? if there is a reason can we put the reason in the recipe

Reply:
We've patched the code to loosen the constraints allowing us to re-instate the `pip check`